components/tikv_util/src/metrics/threads_linux.rs

```diff
@@ -314,6 +314,7 @@ mod tests {
                 f.sync_all().unwrap();
                 tx1.send(()).unwrap();
                 rx.recv().unwrap();
+                fs::remove_file(tmp).unwrap();
```

Comment:
I'm curious why the directory generated by `temp_dir()` doesn't destroyed by it self when the test finished.

Reply:
same question here

Reply:
This `temp_dir` is from `std::env::temp_dir`.